app/models/ocelot/stanzas/CalloutType.scala

```diff
@@ -42,5 +42,20 @@ object CalloutType {
       case JsString("Section") => JsSuccess(Section, __)
       case _ => JsError("Invalid Callout type")
     }
+    
   }
+
+  implicit val writes: Writes[CalloutType] = new Writes[CalloutType] {
```

Comment:
IntelliJ is highlighting trailing whitespace on line 50

Reply:
Ok will strip out